src/main/java/org/spongepowered/api/util/command/CommandCallable.java

```diff
@@ -24,18 +24,19 @@
  */
 package org.spongepowered.api.util.command;
 
+import com.google.common.base.Optional;
 import org.spongepowered.api.text.Text;
-import org.spongepowered.api.util.command.completion.CommandCompleter;
 
 import java.util.List;
 
 /**
- * A command that can be executed.
+ * A low-level interface for commands that can be executed. For almost all use cases, higher-level tools should be used instead, like {@link
+ * CommandSpec}
  *
  * <p>Implementations are not required to implement a sane
- * {@link java.lang.Object#equals(Object)} but may choose to do so.</p>
+ * {@link java.lang.Object#equals(Object)} but really should.</p>
```

Comment:
That doesn't need to be specified explicitly because *Java's* contract for `equals` requires overriding `hashCode` if you override it.